packages/migrate/index.js

```diff
@@ -144,30 +140,22 @@ const Migrate = {
       migrations
     });
 
-    return new Promise((resolve, reject) => {
-      return async.eachSeries(
-        migrations,
-        (migration, finished) => {
-          migration
-            .run(clone)
-            .then(() => {
-              finished();
-            })
-            .catch(error => {
-              finished(error);
-            });
-        },
-        async error => {
-          await this.emitter.emit("postAllMigrations", {
-            dryRun: options.dryRun,
-            error: error ? error.toString() : null
-          });
-
-          if (error) return reject(error);
-          return resolve();
-        }
-      );
-    });
+    try {
+      for (let migration of migrations) {
```

Comment:
```suggestion
      for (const migration of migrations) {
```

just a thought. we might mutate `Migration` so maybe not a good suggestion.